content/blog/cumundi-guest-post/index.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Refactoring Pulumi Code with `aliases`"
+date: 2020-03-26
+meta_desc: "Cumundi uses Pulumi to build infrastructure components for their customers"
+meta_image: cumundi-pulumi.png
+authors:
+    - ringo-de-smet
+tags:
+    - Test Driven Developmnet
+---
+
+**Guest Article:** [Ringo De Smet](https://www.linkedin.com/in/ringodesmet/), Founder of [Cumundi](https://www.cumundi.cloud), standardizes on Pulumi for writing infrastructure as code libraries to deliver reusable building blocks. Pulumi enables him to iterate through the build-test-release cycle of these building blocks rapidly.
+
+Cumundi helps companies adopt the cloud more integrally. Companies are rebuilding the same 75% of the underlying infrastructure. But the shortage of people and time has led to limited investment in integrating all the non-functional requirements needed to run applications in the cloud.
+
+<!--more-->
+
+## How Pulumi Code Keeps Up with Changing Requirements
+
+At Cumundi, we build reusable libraries for our customers to set up their infrastructure integrated with best practices. These best practices span the range of non-functional requirements, which are most of the time not taken up with the same focus as application feature development. Where these non-functional requirements can be integrated into infrastructure setups, we include these in our code libraries. Cumundi focuses on all three major cloud providers to provide application-focused infrastructure libraries.
```

Comment:
```suggestion
At Cumundi, we build reusable libraries for our customers to set up their infrastructure while following best practices. These best practices include a range of "non-functional requirements" - infrastructure patterns - which often receive less focus than application features. Where feasible, Cumundi encapsulates these non-functional requirements into infrastructure code libraries. This enables Cumundi to serve clients using all major clouds while keeping infrastructure libraries clean and modular.
```

Reply:
changed it a bit to keep the intent of application focus more clear